selfdrive/ui/qt/home.cc

```diff
@@ -19,7 +19,7 @@ HomeWindow::HomeWindow(QWidget* parent) : QWidget(parent) {
 
   sidebar = new Sidebar(this);
   main_layout->addWidget(sidebar);
-  QObject::connect(this, &HomeWindow::update, sidebar, &Sidebar::updateState);
+  QObject::connect(uiState(), &UIState::uiUpdate, sidebar, &Sidebar::updateState);
```

Comment:
if we're using uiState(), doesn't it make more sense to do these connections in Sidebar?